### ISSUE TITLE
WCAG CWS Issues

### ISSUE THREAD

**Issue by yileifeng** [ramp4-pcar4/story-ramp#253]
Use [WAVE tool](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) to see multiple WCAG issues that pop up. Mainly, this involves: 

- [ ] images missing alt tages 
- [ ] empty buttons in image panels
